components/fxa-client/src/internal/oauth.rs

```diff
@@ -250,17 +252,12 @@ impl FirefoxAccount {
             scope: auth_params.scope.join(" "),
             state: auth_params.state,
             access_type: auth_params.access_type,
-            code_challenge: auth_params
-                .pkce_params
-                .as_ref()
-                .map(|param| param.code_challenge.clone()),
-            code_challenge_method: auth_params
-                .pkce_params
-                .map(|param| param.code_challenge_method),
+            code_challenge: auth_params.code_challenge,
+            code_challenge_method: auth_params.code_challenge_method,
```

Comment:
I didn't think that having the two PKCE params in a separate sub-struct was really pulling its weight as an abstraction, so I've just made them ordinary fields on the `AuthorizationParams` struct.